src/toil_scripts/lib/programs.py

```diff
@@ -15,7 +15,8 @@ def mock_mode():
     return True if int(os.environ.get('TOIL_SCRIPTS_MOCK_MODE', '0')) else False
 
 
-def docker_call(tool,
+def docker_call(tool='',
```

Comment:
You should never use mutable types as default value for keyword arguments.

It's also unnecessary, tool and tools should default to None and since `bool(None) is False` so you should be good for the tests below.